ecl2df/common.py

```diff
@@ -621,3 +622,46 @@ def stack_on_colnames(dframe, sep="@", stackcolname="DATE", inplace=True):
     del dframe["level_0"]
     dframe.index.name = ""
     return dframe
+
+
+def parse_zonemapfile(filename: str):
+    """Return a dictionary from (int) K layers in the simgrid to strings
+
+    Typical usage is to map from grid layer to zone names.
+
+    The layer filename must currently follow format::
+
+      'ZoneA' 1-4
+      'ZoneB' 5-10
+
+    where the single quotes are optional for zones without spaces.
+    Write single layer zones as 11-11. NB: ResInsight requires single
+    quotes always.
+
+    Args:
+        filename (str): Absolute path to a zone map file (lyr format)
+
+    Returns:
+        dict, integer keys which are the K layers. Every layer mentioned
+            in the interval in the input file is present. Can be empty.
+    """
+    zonelines = Path(filename).read_text().splitlines()
+    zonelines = [line.strip() for line in zonelines]
+    zonelines = [line.split("--")[0] for line in zonelines]
```

Comment:
```suggestion
    zonelines = [line.strip().split("--")[0] for line in Path(filename).read_text().splitlines()]
```
Maybe more readable wrt. operations done?